backend/src/classes/Password.ts

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+
+export default class Password {
+  private static algo = "sha512";
```

Comment:
you should use `argon2` as your hashing algorithm as its better suited to passwords